agent/util-scripts/get-internal-tool

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env python3
+# -*- mode: python -*-
+
+import sys
+import json
+from pathlib import Path
+
+if len(sys.argv) != 4:
+    print("Usage: get-internal-tool <metadata_location> <tool_name> <tool_type>")
+    sys.exit()
+
+meta_loc = Path(sys.argv[1])
+name = sys.argv[2]
+tool_type = sys.argv[3]
```

Comment:
I realize this is a small and relatively simple tool; but it'd be easier to read and maintain with `ArgumentParser` all the same.